api/src/controllers/changes.js

```diff
@@ -399,6 +408,17 @@ const request = (req, res) => {
   });
 };
 
+const logWarnings = () => {
+  if (Object.keys(docCountUserWarnings).length) {
```

Comment:
I think we can drop the `if` here - the loop will do nothing if the length is 0.

Reply:
done